src/components/layout/Center/Center.tsx

```diff
@@ -65,12 +66,14 @@ Center.propTypes = {
   gutters: PropTypes.oneOf(Object.values(SpaceSizes)),
   areChildrenCentered: PropTypes.bool,
   isTextCentered: PropTypes.bool,
+  className: PropTypes.string,
 };
 
 Center.defaultProps = {
   gutters: SpaceSizes.none,
   areChildrenCentered: false,
   isTextCentered: false,
+  className: CLX_LAYOUT,
```

Comment:
We cannot do it this way because default props are overridden from outside, you need to do it like: 
```ts
const Center = styled.div
  .withConfig({
    shouldForwardProp: (prop) =>
      !includes(prop, [
        'maxWidth',
        'gutters',
        'areChildrenCentered',
        'isTextCentered',
      ]),
  })
  .attrs((props) => ({
    ...props,
    className: classNames(CLX_LAYOUT, props.className),
  }))<CenterProps>